frontend/react/src/components/ButtonLoader.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { Circles } from "react-loader-spinner";
```

Comment:
Did you import `react-loader-spinner`?

Reply:
yes i imported it. Also for `package-lock.json ` i deleted when i was having issues installing the project, i did not do any modifications.

Reply:
> yes i imported it. Also for `package-lock.json ` i deleted when i was having issues installing the project, i did not do any modifications.

It seems you forgot to push your updated `package.json`. You can also check your changed files [here](https://github.com/AmanNegi/FreshNest/pull/130/files).

Reply:
> > yes i imported it. Also for `package-lock.json ` i deleted when i was having issues installing the project, i did not do any modifications.
> 
> It seems you forgot to push your updated `package.json`. You can also check your changed files [here](https://github.com/AmanNegi/FreshNest/pull/130/files). Also, you're supposed to work on the same `package-lock.json`, otherwise it will lead to conflicts between dependencies. Please feel free to mention what issues you were having.

no file change in `package.json` that was why it was not updated  you are free to merge. i also i didn’t touch the `package-lock.json` i only worked on the files needed to be worked on.

Reply:
@ayobamiseun If you use any external package like in your case `react-loader-spinner` it gets added to package.json when you run `npm install react-loader-spinner`. Anyway, in the loading case, I would suggest you use DaisyUI Loading as we are already using DaisyUI.

- https://daisyui.com/components/loading/#loading-spinner

Regarding, `package-lock.json` unstage it and even if you have any changes in it don't push them.

Reply:
@AmanNegi  how can i unstage the `package-lock.json` its already  part of my first commit.

Reply:
You can simply replace it with the old file, and push it. Because you were facing issues with it, I've fixed and pushed that to your branch.